lib/brain.js

```diff
@@ -812,6 +817,13 @@ Brain.prototype._setState = function _setState (state, oldState) {
   return true
 }
 
+Brain.prototype._factoryReset = function _factoryReset () {
+  factoryResetHelper.nuke(this.dataPath)
+    .then(() => this.trader.unpair())
+    .then(() => this._transitionState('virgin'))
+    .catch(err => console.log('failure factory resetting machine', err))
```

Comment:
I forgot to add the TODO comment on here. This is a little tricky because of the state of l-m depending when the error happened. 

If during nuke the machine will be at a unknown state. (fs code but I think we'll see errors happening there).

If during trader.unpair()  the machine will be virgin again, but the server will have a orphan record of it.

Should I add the failure screen on nuke and going to `virgin` if the failure on unpair?

Reply:
Sounds good.